astropy/stats/tests/test_funcs.py

```diff
@@ -236,6 +238,60 @@ def test_bootstrap():
         bootresult = np.mean(funcs.bootstrap(bootarr, 10000, bootfunc=np.mean))
         assert_allclose(np.mean(bootarr), bootresult, atol=0.01)
 
+    # test a bootfunc with several output values
+    # return just bootstrapping with one output from bootfunc
+    with NumpyRNGContext(42):
+        bootarr = np.array([[1, 2, 3, 4, 5, 6, 7, 8, 9, 0],
+                            [4, 8, 8, 3, 6, 5, 2, 8, 6, 2]]).T
+
+        answer = np.array((0.19425, 0.02094))
+
+        bootresult = funcs.bootstrap(bootarr, 2,
+                                     bootfunc=spearmanr)
```

Comment:
I think you may want to put this new part in a separate function as it depends on scipy. The old test can run without scipy, and this function gets the decorator and runs only when scipy is available.